RELEASE-NOTES.txt

```diff
@@ -1,3 +1,8 @@
+4.2
+-----
+- Products: now the keyboard pop up automatically when a view with only one text field is opened.
```

Comment:
nit: it might be helpful mentioning the screens with the improvement and the milestone (like Edit Product Description in M1)?